tests/parsers/sqlite_plugins/skype.py

```diff
@@ -21,9 +21,10 @@ def testProcess(self):
     """Tests the Process function on a Skype History database file.
 
       The History file contains 24 events:
-      4 call events
+      3 call events
```

Comment:
Any indication of why this changed?

Reply:
As far as I can tell, this comment was never correct.